tests/test_dfa.py

```diff
@@ -1452,4 +1452,230 @@ def test_dfa_repr(self):
         )
         self.assertEqual(
             repr(dfa),
-            "DFA(states={'q0'}, input_symbols={'a'}, transitions={'q0': {'a': 'q0'}}, initial_state='q0', final_states={'q0'}, allow_partial=False)")  # noqa: E501
+            "DFA(states={'q0'}, input_symbols={'a'}, transitions={'q0': {'a': 'q0'}}, "
+            "initial_state='q0', final_states={'q0'}, allow_partial=False)")
+
+    def test_iter_finite(self):
+        """
+        Test that DFA for finite language generates all words
+        """
+        language = {'aa', 'aaa', 'aaba', 'aabbb', 'abaa', 'ababb', 'abbab',
+                    'baa', 'babb', 'bbaa', 'bbabb', 'bbbab'}
+        dfa = DFA.from_finite_language(language, {'a', 'b'})
+        generated_set = {word for word in dfa}
+        self.assertEqual(generated_set, language)
+
+    def test_iter_infinite(self):
+        """
+        Test that language that avoids the pattern '11' generates the correct values in correct order
+        """
+        A = DFA(
```

Comment:
@Tagl Could you please rename `A` to something like `dfa` or `dfa1`? The capital letter makes me think `A` is a class rather than an instance.

Reply:
Will do

Reply:
Updated

Reply:
Thanks, although I think there's still one hanging around:

<img width="734" alt="Screenshot 2022-10-27 at 1 42 56 PM" src="https://user-images.githubusercontent.com/872474/198393937-f1c5adfb-5fcd-46e3-8ddd-fa4519a6eb62.png">

Reply:
Missed one originally, updated again.